nyc_taxis/TEST_PROCEDURES.md

```diff
@@ -0,0 +1,55 @@
+# Test Procedures
+
+## searchable_snapshot 
+
+A test procedure for measuring performance for Searchable Snapshots feature. It runs the same search query as the default test procedure `append-no-conflicts`.
+
+In contrast with `append-no-conflicts` which running query on index with local storage, this test procedure runs query on index backed by a remote snapshot.
```

Comment:
which runs queries on an  index stored on the cluster itself, this test procedure runs queries on an index ...